server/src/test/java/org/elasticsearch/search/aggregations/bucket/nested/NestedAggregatorTests.java

```diff
@@ -87,6 +87,21 @@
 import static org.elasticsearch.search.aggregations.AggregationBuilders.nested;
 import static org.hamcrest.Matchers.equalTo;
 
+/**
+ * Tests for the Nested aggregator.
+ *
```

Comment:
`<p>`.

Silly javadoc, you are too html.

Reply:
Bad HTML.  `<p>` != `<br/>`.  But fine.

Reply:
? I mean, you could `<p>` the whole paragraph. I dunno. whatever works.